### ISSUE TITLE
Make URL paths more distinctive

### ISSUE THREAD

**Issue by krista-ramakers** [driebit/ginger#259]
As analysing pageviews becomes more and more important for our clients, the question has risen whether it's possible to include, for example, the page category or content group in the URL. This might have multiple advantages:
- It is easier to tell URL's apart and check if you are measuring the performance of the right page
- It is easier to exclude/filter on specific pages in analytics (since you can only filter on (part of) URL's)

Can we discuss a possible solution for this issue? I expect this will only become more relevant in the future, for other customers too.

**Comment by Dirklectisch** [comment]
I'm not sure if this is a good idea. It would change every existing URL in the site. I think Google Analytics has other ways of segmenting page views that we could use?

**Comment by ddeboer** [comment]
> It would change every existing URL in the site. 

Right. Additionally, we must prevent page URLs changing when their category is changed. While we can solve this by adding the category *after* the id, like `http://example.com/en/page/346/text/groot-worden-in-een-grote-stad`, that looks rather ugly.

And is unnecessary. A simpler solution is just to change the `ga('send', 'pageview', ...)` calls to send additional details, such as the category in the URL. Google calls this [virtual pageviews](https://developers.google.com/analytics/devguides/collection/analyticsjs/single-page-applications#tracking_virtual_pageviews) (for tracking SPAs).

**Comment by krista-ramakers** [comment]
Sounds good! But is it comprehensive enough for the users of GA/Tag manager? How they can access the data of the specific pages they need?
